### PR TITLE
fix reprovisioning failure after failing to provision once

### PR DESCRIPTION
**Pull request by rahul-daga-imgtec**
Signed-off-by: Rahul Daga <Rahul.Daga@imgtec.com>

### CODE REVIEW

File: src/fdm_register.c [hunk 1]
```diff
@@ -291,7 +291,7 @@ bool DefineObjectsAtClient(AwaClientSession *session, const OBJECT_T *objects,
  * @return true for success otherwise false.
```
Comment (pratik-prajapati-imgtec) [inline src/fdm_register.c]: add description for new param

**Comment by nikhil-zinjurde-imgtec** [comment]
Blocked on verifying this due to boot from USB and SD Card both currently broken

**Comment by abhijit-mahajani-imgtec** [comment]
you can always use ethboot. it works very well.

**Comment by abhijit-mahajani-imgtec** [comment]
Verified.

**Comment by abhijit-mahajani-imgtec** [comment]
Approved.

**Pull request merged**